src/store.js

```diff
@@ -94,7 +94,11 @@ class Store {
     }
 
     if (_.isArray(data)) {
```

Comment:
The original PR permitted the meta attribute on models as well as collections. I'm sure we can do a follow-up PR if we need that functionality, but curious to know your thoughts on whether that might be applicable in the future. So far, I only know of meta being used on the product search endpoint.

Reply:
Yeah, I chatted with Tyler about that. There are some caching issues with allowing meta on models. With Stas' implementation, meta only occurs for the initial fetch. So, if we want model meta data, we need to use `store.fetch`. As Noah pointed out, this also means the objects returned from `store.get` and `store.fetch` will differ, as the meta is not cached on the model. If the metadata is cached, then we're probably getting stale meta data when we get cached responses. So then the question is, do we need meta data on models? The answer for now is no, but happy to discuss.

Reply:
Talked with stephen about this offline, he clarified collections aren't cached so we need to don't worry about whether or not we should cache the meta value. There seems to be some trickiness about whether we'd have to cache meta as part of the model or outside of the model -- if we do the latter, we might not get the meta value in the cached response later (that's what I understood anyway, could be off base!).